### DISCUSSION TITLE
Minimum match score option

### DISCUSSION THREAD

**Discussion by Serene-Arc** [beetbox/beets#4527] [Ideas]
Just an idea that would be really helpful for me: when importing, either import or ask for confirmation if the match threshold is above a certain number (say 50%) and otherwise skips it, as with the regular `-q` option.

This is basically an extension of the '-q' option where beets skips all unmatched things but you can pair it with `-t` or whatever to tag everything that's kinda close.

**Comment by sampsyo** [comment]
You can do that by setting `none_rec_option` to `skip`:
https://beets.readthedocs.io/en/stable/reference/config.html#none-rec-action

You can also adjust `medium_rec_thresh` to control the similarity threshold at which this happens:
https://beets.readthedocs.io/en/stable/reference/config.html#autotagger-matching-options

**Reply by Serene-Arc** [reply]
Ah, thanks! I wondered if I was missing something in the options, thank you!

Side note: are there any plans for a release soon?
